### ISSUE TITLE
Reproduce Problem

### ISSUE THREAD

**Issue by Camellia-tx** [lucidrains/CoCa-pytorch#19]
Thanks for this repo. I'm using my own dataset for pre-training via CoCa, and I found that the contrastive loss output from each batch is basically unchanged, is it the contrastive_label that needs to be changed or is there some other place where I need to make corresponding changes?
Thank you!!!!

**Comment by lucidrains** [comment]
the code in this repo has been used to reproduce [an open sourced model already.](https://laion.ai/blog/coca/)

**Comment by Camellia-tx** [comment]
I apologize for only replying now, thank you so much for your contribution!
















在 2023-10-27 23:30:19，"Phil Wang" ***@***.***> 写道：

Closed #19 as completed.
